.github/scripts/stats/fetch_update_gist.sh

```
@@ -0,0 +1,14 @@
+CLONES=$(curl -s -H "Authorization: token ${{ secrets.LIGHTVM_PAT }}" \
+"https://api.github.com/repos/${{ github.repository }}/traffic/clones" \
+| jq '.count // 0')
+
+TRAFFIC_DATA=$(curl -s -H "Authorization: token ${{ secrets.LIGHTVM_PAT }}" \
+"https://api.github.com/repos/${{ github.repository }}/traffic/views")
+
+TOTAL_VIEWS=$(echo $TRAFFIC_DATA | jq '.count // 0')
+UNIQUE_VISITORS=$(echo $TRAFFIC_DATA | jq '.uniques // 0')
+
+curl -s -X PATCH -H "Authorization: token ${{ secrets.LIGHTVM_PAT }}" \
```

Shellcheck (0.11.0)

[error] Parameter expansions can't start with {. Double check syntax.

(SC2296)